src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
I don't use require.js, but quickly tried it with this change. @Emigre you are right that OpenSeadragon will leak to global scope. I'll try to see how one can fix this properly.